tests/kernel/nvmftests.pm

```diff
@@ -0,0 +1,82 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 Michael Moese <mmoese@suse.de>
+# Copyright © 2012-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: new test that runs NVMe over Fabrics testsuite
+# Maintainer: Michael Moese <mmoese@suse.de>
+
+use base 'opensusebasetest';
+use strict;
+use testapi;
+use utils 'zypper_call';
+
+sub run {
+    select_console('root-virtio-terminal');
```

Comment:
take a look at  `select_virtio_console` and it's usages , let's use same flow for virtio console